src/analysis/special_functions/pow.lean

```diff
@@ -1248,6 +1264,44 @@ lemma rpow_le_rpow_of_exponent_ge {x : ℝ≥0} {y z : ℝ} (hx0 : 0 < x) (hx1 :
   x^y ≤ x^z :=
 real.rpow_le_rpow_of_exponent_ge hx0 hx1 hyz
 
+lemma le_neg_rpow_iff {a b c : ℝ} (ha : 0 < a) (hb : 0 < b) (hc : 0 < c) :
+  a ≤ b^(-c) ↔ b ≤ a^(-c⁻¹) :=
```

Comment:
This feels like it might be more convenient as
```suggestion
lemma le_neg_rpow_iff {a b c : ℝ} (ha : 0 < a) (hb : 0 < b) (hc : c < 0) :
  a ≤ b^c ↔ b ≤ a^c⁻¹ :=
```

Reply:
I tend to disagree with you here. Since we have `positivity` it is way easier to deal with positive numbers then with negative numbers.

Reply:
But what are the chances that the LHS or RHS will show up, vs those in Eric's suggestion. His version is syntactically more useful. If anything, we should teach `positivity` to turn `-c < 0` into `0 < c`.

Reply:
The example is in the linked PR. If you change `r` to `-r` there, then the condition for integrability becomes either `finrank + r < 0` which is strange to work with. You usually use that proposition by artifically introducing a factor `1 = (1 + |x|)^r (1 + |x|)^(-r)`, for `finrank < r` and then split the integral. There the negative of a positive number appears very naturally.

Reply:
But you don't need to change that example, right? Just this lemma. If you keep that example as is, then this lemma should still work. Or am I missing something?